product_hunt_utils.py

```diff
@@ -1,84 +1,131 @@
+    'first_visit': '1691049905',
+    'first_referer': 'https://www.google.com/',
+    '_hjHasCachedUserAttributes': 'true',
+    'intercom-session-fe4ce68d4a8352909f553b276994db414d33a55c': '',
+    'analytics_session_id': '1691513461352',
+    '_hjAbsoluteSessionInProgress': '1',
+    '_hjIncludedInSessionSample_3508551': '0',
+    '_hjSession_3508551': 'eyJpZCI6ImYzYzRlZWY1LTk2ZjUtNGY0Mi1iMGI1LWRjM2M3OTNkNjE3MyIsImNyZWF0ZWQiOjE2OTE1MTM0NjY1NTUsImluU2FtcGxlIjpmYWxzZX0=',
+    'csrf_token': 'W4IQ0p2YGhOSVtgMvGBRh0hLmnqHjLR4__Zd80swYot9dbxQT5s2VPS8h0svazUhapvRws3h_6-FBqE8IVmYlw',
+    '_producthunt_session_production': 'OSf3xKPYByDwwW3YgyA0lXpmXswQe7M%2FwsyrNOLSNPgj1kgRzt2p9urAI8NjiqPKcC45uzm98eZ5K8v43wxVBsdtjMoJ8HOkh5WaAo%2Fv8z00LQ2fax0lIzyrQLVE7hbl%2BmE7MJ4z7qVq9pEFF7uS4uYTdYQD9y8Ehzvc0W%2B8big%2B%2BxNih%2FSMf%2FdtO5mvWUcNVHH9JGGeo1p0EkNorUp7ArowwwPpqjeuxqEUy1dp0jr7g1Wr8sHBZdLM55gkC1nNAmYjf4kNB6g2d0uKMZhfdordK2e2IFaUbpuZrEf%2F6BBMYiwCPz9lopb1WjjovkuqgldCYuhJhGu5O6rj%2FKtRiOFuzcqQAO6085CFgtvMFlvdxTdtj5DwDVOpJV78V7dWn0jpu%2BcdZ5FSdlEDrs0Zx716QxOaKbtNnHWr1OaNPKIp6Ft4JJ8i5rZ5UgWsStlw4nPPoTXWQNqBrbC1wCu%2BwNOyk1wlaYg8EZ%2B5wVfEg2CAmLRN2n%2FTi7qlR5gpkYwnv7igrWBtmacJY8GziQ%3D%3D--SfIMKCTzQmgIiVkM--hbRy8HbB9c8bEVneYjHwpQ%3D%3D',
+    'analytics_session_id.last_access': '1691513931850',
+    '_ga_WZ46833KH9': 'GS1.1.1691513462.24.1.1691513932.60.0.0',
+}
+
 headers = {
-            'authority': 'www.producthunt.com',
-            'accept': '*/*',
-            'accept-language': 'en-US,en;q=0.9',
-            'content-type': 'application/json',
-            'origin': 'https://www.producthunt.com',
-            'referer': 'https://www.producthunt.com/@cyriljeet',
-            'sec-ch-ua': '"Not/A)Brand";v="99", "Google Chrome";v="115", "Chromium";v="115"',
-            'sec-ch-ua-mobile': '?0',
-            'sec-ch-ua-platform': '"Windows"',
-            'sec-fetch-dest': 'empty',
-            'sec-fetch-mode': 'cors',
-            'sec-fetch-site': 'same-origin',
-            'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36',
-            'x-requested-with': 'XMLHttpRequest',
-        }
+    'authority': 'www.producthunt.com',
+    'accept': '*/*',
+    'accept-language': 'en-US,en;q=0.9',
+    'content-type': 'application/json',
+    'origin': 'https://www.producthunt.com',
+    'referer': 'https://www.producthunt.com/@emmanuelonuoha',
+    'sec-ch-ua': '"Not/A)Brand";v="99", "Google Chrome";v="115", "Chromium";v="115"',
+    'sec-ch-ua-mobile': '?0',
+    'sec-ch-ua-platform': '"Windows"',
+    'sec-fetch-dest': 'empty',
+    'sec-fetch-mode': 'cors',
+    'sec-fetch-site': 'same-origin',
+    'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36',
+    'x-requested-with': 'XMLHttpRequest',
+}
 
-def get_commentor_info_for_product_id(product_id: int)-> list:

```

Comment:
where is `get_commentor_info_for_product_id` function?

Reply:
actually i did not  pushed whole code , just the following part